src/edu/pugetsound/mathcs/nlp/architecture_nlp/features/FoldCreator.java

```diff
@@ -0,0 +1,211 @@
+package edu.pugetsound.mathcs.nlp.architecture_nlp.features;
+import java.io.*;
+
+import java.util.Random;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Scanner;
+
+import com.sun.xml.internal.bind.v2.runtime.unmarshaller.XsiNilLoader.Array;
+
+import edu.pugetsound.mathcs.nlp.lang.Conversation;
+import edu.pugetsound.mathcs.nlp.lang.Utterance;
+import edu.pugetsound.mathcs.nlp.kb.KBController;
+import edu.pugetsound.mathcs.nlp.architecture_nlp.features.TextAnalyzer;
+
+/**
+ * 
+ * Contains a main method which takes 2 files, one full of words and their parts of speech, and another with 
+ * sentences and their logical parses. Together it creates a new file with new sentences from the word file paired
+ * with the logical parses from the other. 
+ * REQUIRES A words.txt and test.ccg files to operate. 
+ *  
+ * @author NathanielLivingston
+ */
+// pull all code put in spf read in geo file push out fold push request keep verb the same!
+
+
+public class FoldCreator{ 
+
+	/**
+	 * @param args
+	 */
+	public static void main(String [] args)
+	{
+		System.out.println(System.getProperty("user.dir"));
+		try {
+			
+			String line = null;
+			
+			FileReader fileReader = new FileReader("src/edu/pugetsound/mathcs/nlp/architecture_nlp/features/words.txt"); // open the word file
+			FileReader fileReader2 = new FileReader("src/edu/pugetsound/mathcs/nlp/architecture_nlp/features/test.ccg"); // open the fold file
+
+			BufferedReader bufferedReader = new BufferedReader(fileReader);
+			BufferedReader bufferedReader2 = new BufferedReader(fileReader2);
+
+			
+			ArrayList<String> newWords = new ArrayList<String>(); // create arrayLists for both
+			ArrayList<String> atlas = new ArrayList<String>();
+
+			
+            while((line = bufferedReader.readLine()) != null) // fill the wordlist
+            	{
+	            	newWords.add(line); 
+            	} 
+            
+            while((line = bufferedReader2.readLine()) != null) // fill the atlas full of the ccg data
+	        	{
+	            	atlas.add(line);
+	        	} 
+
+            
+            BufferedWriter writer = new BufferedWriter(new FileWriter("newTest")); // prepare to write to a file
+	        
+            for(int t = 0; t < atlas.size(); t=t+3) // for each sentence in the ccg data   
+	            
+	        {
+	        	
+	    		KBController KB = new KBController("."); // prepare to analyze
+	    		TextAnalyzer TA = new TextAnalyzer(KB);
+	    		Conversation talk = new Conversation();
+	    		
+	    		
+	        	System.out.println("attempting to analyze sentance-> "+ atlas.get(t));
```

Comment:
There should be no stray print statements. Any print statements should be wrapped in an if-statement:
if(Logger.debug()){
//print here
}
The Logger class is also in the util directory